integration/query_comparision_test.go

```diff
@@ -285,6 +293,10 @@ func TestQueryComparisonEq(t *testing.T) {
 			filter:      bson.D{{"value", bson.D{{"$eq", int32(math.MinInt32)}}}},
 			expectedIDs: []any{"int32-min"},
 		},
+		"Int32DoubleBig": {
+			filter:      bson.D{{"value", bson.D{{"$eq", float32(2 << 25)}}}},
```

Comment:
`float32` is not a valid BSON type; driver converts it under then hood, so there is no need to test it

Reply:
Removed